app.json

```diff
@@ -1,37 +1,38 @@
 {
   "expo": {
     "name": "bathroom-finder-frontend",
+    "scheme": "bathroom-finder-frontend",
     "slug": "bathroom-finder-frontend",
     "version": "1.0.0",
-    "orientation": "portrait",
-    "icon": "./assets/icon.png",
-    "userInterfaceStyle": "light",
-    "splash": {
-      "image": "./assets/splash.png",
-      "resizeMode": "contain",
+    "orientation": "portrait"
+  },
+  "icon": "./assets/images/icon.png",
+  "userInterfaceStyle": "light",
+  "splash": {
+    "image": "./assets/images/splash.png",
+    "resizeMode": "contain",
+    "backgroundColor": "#ffffff"
+  },
+  "assetBundlePatterns": ["**/*"],
+  "ios": {
+    "supportsTablet": true
+  },
+  "android": {
+    "adaptiveIcon": {
+      "foregroundImage": "./assets/images/adaptive-icon.png",
       "backgroundColor": "#ffffff"
-    },
-    "assetBundlePatterns": ["**/*"],
-    "ios": {
-      "supportsTablet": true
-    },
-    "android": {
-      "adaptiveIcon": {
-        "foregroundImage": "./assets/adaptive-icon.png",
-        "backgroundColor": "#ffffff"
+    }
+  },
+  "web": {
+    "favicon": "./assets/images/favicon.png"
+  },
+  "plugins": [
+    [
+      "expo-font",
+      {
+        "fonts": ["./assets/fonts/EudoxusSans-Regular.ttf"]
       }
-    },
-    "web": {
-      "favicon": "./assets/images/favicon.png"
-    },
-    "plugins": [
-      [
-        "expo-font",
-        {
-          "fonts": ["./assets/fonts/EudoxusSans-Regular.ttf"]
-        }
-      ]
-    ],
-    "platforms": ["ios", "android"]
-  }
+    ]
+  ],
+  "platforms": ["ios", "android"]
```

Comment:
That platforms list is what keeps web from compiling. A curly brace dissapeared under "platforms". It's likely that you unintentionally broke the config file. Please go back over your changes to this file and make sure that you only change what is absolutely necessary

Reply:
Yep that was it which is annoying. I never touched that file but the format did.